aptos-move/aptos-vm/src/storage_adapter/executor_view_base.rs

```diff
@@ -44,32 +44,39 @@ impl<S: StateView> AsExecutorView<S> for S {
     }
 }
 
-impl<'s, S: StateView> TDelayedFieldView for ExecutorViewBase<'s, S> {
-    type IdentifierV1 = StateKey;
-    type IdentifierV2 = DelayedFieldID;
+impl<'s, S: StateView> TAggregatorV1View for ExecutorViewBase<'s, S> {
+    type Identifier = StateKey;
 
     fn get_aggregator_v1_state_value(
         &self,
-        state_key: &Self::IdentifierV1,
+        state_key: &Self::Identifier,
         // Reading from StateView can be in precise mode only.
     ) -> anyhow::Result<Option<StateValue>> {
         self.base.get_state_value(state_key)
     }
+}
+
+impl<'s, S: StateView> TDelayedFieldView for ExecutorViewBase<'s, S> {
+    type Identifier = DelayedFieldID;
+
+    fn is_aggregator_v2_delayed_fields_enabled(&self) -> bool {
+        false
```

Comment:
The flag is hardcoded here? We earlier used to have feature flags AGGREGATOR_V2_API_ENABLED, AGGREGATOR_V2_EXECUTION_ENABLED right?